.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:8.11
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
+          # fallback to using the latest cache if no exact match is found
```

Comment:
Why is this desirable?

Reply:
Caching `node_moduels` in between builds can speed up testing time quite a bit for larger projects.  When you use yarn locally on your computer it keeps a cache in a folder on your hard-drive.  I'm guessing since they run your build/test in a random container at circle it doesn't have a local disk cache and so without some extra caching mechanism like this every time you ran tests it would do an install of all package.json dependencies over the network, which _can_ be slow.

Reply:
fwiw I don't know all the details - this file is a literal copy & paste from their instructions when you add a new repo.